composer/modules/web/src/plugins/text-file-editor/plugin.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import Plugin from 'core/plugin/plugin';
+import { CONTRIBUTIONS } from 'core/plugin/constants';
+import TextEditor from './editor/TextEditor';
+
+// extensions to support
+const exts = ['xml', 'json', 'toml', 'conf', 'css', 'html', 'txt', 'yml', 'md', 'js',
```

Comment:
We might have to review which files to support, probably files you need for day-to-day integration scenarios.